docs/man-openmpi/man3/MPIX_Query_cuda_support.3.rst

```diff
@@ -40,33 +40,58 @@ There is no C++ binding for this function.
 DESCRIPTION
 -----------
 
-This routine return 1 if MPI library is build with CUDA and runtime
-supports CUDA buffers. This routine must be called after MPI is
-initialized by a call to :ref:`MPI_Init` or :ref:`MPI_Init_thread`.
+This function is part of an :ref:`Open MPI extension
+<ompi-features-extentions-label>`; it is not part of standard MPI.
+
+This routine returns 1 if both the MPI library was built with the AMD
```

Comment:
This should probably by Nvidia here (not AMD)

Reply:
🤬 The perils of copy-n-paste... Thanks for catching.